index.ts

```diff
@@ -81,6 +83,10 @@ interface ClaspSettings {
   expiry_date: string;
 }
 // Project settings file (Saved in .clasp.json)
+interface ClaspProjectSettings {
```

Comment:
`ClaspProjectSettings` is not needed. How is it different than `ProjectSettings`?

Reply:
`ClaspProjectSettings` is object to enclose `ProjectSettings` and has multiple `ProjectSettings`.
In https://github.com/google/clasp/issues/12#issuecomment-370672417, `ClaspProjectSettings` equivalent to `ProjectSettings` and `ProjectSettings` equivalent to `Env`.

Reply:
Can you use the exact terms as in the comment? I wrote:

```ts
// Project settings file (Saved in .clasp.json)
interface ProjectSettings {
  scriptId: string; // default env
  rootDir: string;  // default env
  envs: Map<EnvId, Env> // custom scriptIds (NEW)
}
```

`ClaspProjectSettings` is redundant. Either say `ClaspSettings` or `ProjectSettings`.

Reply:
I see. I will use extact terms.